code/script.js

```diff
@@ -0,0 +1,81 @@
+const projectCard = document.getElementById('projectCard')
+const profile = document.getElementById('profile')
+const profileImage = document.getElementById('profileImage')
+
+// Github Profile
+  fetch('https://api.github.com/users/rawisou')
+   .then((res) => res.json()) 
+   .then((data) => {
+    profile.innerHTML = `
```

Comment:
You should declare `profile` here instead of in the global scope. 

To take it one step further, an alternative is to not actually assign a variable since there is only this one occurrence when you need to access this element. So instead of:
```javascript
const profile = document.getElementById('profile')
profile.innerHtml = ...
```
You could directly write:
```javascript
document.getElementById('profile').innerHtml = ...